src/App.tsx

```diff
@@ -1,7 +1,114 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useLocalStorage } from './UseLocalStorage';
+import { Todolist } from './Todolist';
+import { Todo } from './types/todo';
+import { TodoFilter } from './TodosFilter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useLocalStorage<Todo[]>('todos', []);
```

Comment:
it's useful to add some enum where you can store all you local storage keys, because then you can easily change some key, or add the new one